common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3273,11 +3273,11 @@ public static enum ConfVars {
 
     HIVE_COMPACTOR_ABORTEDTXN_THRESHOLD("hive.compactor.abortedtxn.threshold", 1000,
         "Number of aborted transactions involving a given table or partition that will trigger\n" +
-        "a major compaction."),
+        "a major compaction / cleanup of aborted directories."),
 
     HIVE_COMPACTOR_ABORTEDTXN_TIME_THRESHOLD("hive.compactor.aborted.txn.time.threshold", "12h",
         new TimeValidator(TimeUnit.HOURS),
-        "Age of table/partition's oldest aborted transaction when compaction will be triggered. " +
+        "Age of table/partition's oldest aborted transaction when compaction / cleanup of aborted directories will be triggered. " +
```

Comment:
same as above

Reply:
This config is used for controlling the time threshold of aborted txns after which the cleanup is triggered (either cleanup / compaction) depending on what is expected.